.travis.yml

```diff
@@ -14,7 +17,7 @@ env:
     - PGVERSION=9.5
     - PGVERSION=9.6
 before_install:
-  - git clone -b v0.6.1 --depth 1 https://github.com/citusdata/tools.git
+  - git clone -b use_cached_postgresql_build --depth 1 https://github.com/citusdata/tools.git
```

Comment:
After merging [this PR](https://github.com/citusdata/tools/pull/16), we will change this line to clone correct tag.